pravega-provisioner/cluster_provisioner.py

```diff
@@ -161,25 +183,40 @@ def resource_based_provisioning(vms, vm_cpus, vm_ram_gb, vm_local_drives, zookee
     # in-memory cache size for a Segment Store would be as follows:
     min_vm_mem_available = min(mem for (cpu, mem, disks, processes_in_vm) in the_cluster)
     max_segment_stores_per_vm = math.ceil(segment_stores / vms)
-    new_direct_memory = Constants.segment_store_direct_memory_in_gb + int(min_vm_mem_available / max_segment_stores_per_vm)
-    print("--------- Segment Store In-Memory Cache Size (Pravega +0.7) ---------")
-    print("Segment Store pod memory limit: ", Constants.segment_store_jvm_size_in_gb + new_direct_memory, "GB")
-    print("Segment Store JVM Size (-Xmx JVM Option) : ", Constants.segment_store_jvm_size_in_gb, "GB")
-    print("Segment Store direct memory (-DirectMemory JVM Option): ", new_direct_memory, "GB")
-    # We leave 1GB extra of direct memory for non-caching purposes
-    print("Segment Store cache size (pravegaservice.cacheMaxSize): ", (new_direct_memory - 1) * 1024 * 1024 * 1024,
-          " (", new_direct_memory - 1, "GB)")
-    print("Buffering time that Pravega tolerates with Tier 2 unavailable given a (well distributed) write workload:")
-    for w in range(100, 1000, 200):
-        print("- Write throughput: ", w, "(MBps) -> buffering time: ", ((segment_stores * (new_direct_memory - 1) * 1024) / w), " seconds")
+    new_direct_memory = 0 
+    if max_segment_stores_per_vm > 0:
```

Comment:
`max_segment_stores_per_vm` should be always > 0, given that if not the program will exit in the previous assertion that checks for the allocation of a "minimal cluster" (that requires at least 1 segment store). So you can remove this if block.